### ISSUE TITLE
[Vulnerability] Prototype pollution

### ISSUE THREAD

**Issue by aldulea** [pattern-lab/patternlab-node#990]
```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ patternlab-node                                              │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ patternlab-node > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Please bump `lodash` to required version.

**Comment by ronalson** [comment]
I'm getting the same warning:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ High          │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.11                                                    │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ patternlab-node [dev]                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ patternlab-node > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/782                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Low           │ Prototype Pollution                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ lodash                                                       │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=4.17.5                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ patternlab-node [dev]                                        │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ patternlab-node > lodash                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://nodesecurity.io/advisories/577                       │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

Following the issue on [lodash](https://github.com/lodash/lodash) repo it's possible to see that a solution was publish today at version `lodash@4.17.12`: https://github.com/lodash/lodash/issues/4348

**Comment by simonerlandsen** [comment]
does anyone have a way to solve this? making lodash a direct dep, deleting node_modules folder and package-lock and reinstalling deps usually helps in these situations, but not for this.
